rdmd.d

```diff
@@ -828,11 +828,20 @@ Returns:
 */
 string innerEvalCode(string[] eval)
 {
+    import std.conv : text;
     import std.string : join, stripRight;
     // assumeSafeAppend just to avoid unnecessary reallocation
     string code = eval.join("\n").stripRight.assumeSafeAppend;
+    auto lastSemicolon = code.lastIndexOf(";");
     if (code.length > 0 && code[$ - 1] != ';')
-        code ~= ';';
```

Comment:
FYI: this change comes from https://github.com/dlang/tools/pull/303 which hasn't been released yet.